src/connectionhealthpolicy/ConnectionHealthPolicyConfiguration.ts

```diff
@@ -39,4 +42,15 @@ export default class ConnectionHealthPolicyConfiguration {
    * as the microphone may sometimes cause a delay in sending audio packets during the initial stages of a connection.
    */
   sendingAudioFailureInitialWaitTimeMs = 3000;
+
+  /**
+   * Policies and parameters related to video encoding health montoring
+   */
+  videoEncodingHealthPolicies = [
+    VideoEncodingCpuConnectionHealthPolicy,
+    VideoEncodingFramerateConnectionHealthPolicy,
+  ];
+  consecutiveHighEncodeCpuThreshold = 10;
```

Comment:
could these have a short description + effects of increasing or decreasing?

Reply:
Added comments to describe effects for these parameters.